cosmos-gui/src/cmd_runner.js

```diff
@@ -28,18 +28,19 @@ var spawn = require('child_process').spawn;
 
 function run(cmd, params, callback) {
     var execution = spawn(cmd, params);
-    var result = '';
+    var resStdout = '';
+    var resStderr = '';
 
     execution.stdout.on('data', function (data) {
-        result += 'stdout: ' + data.toString();
+        resStdout += data.toString();
     });
 
     execution.stderr.on('data', function (data) {
-        result += 'stderr: ' + data.toString();
+        resStderr += + data.toString();
     });
 
     execution.on('close', function (code) {
-        return callback(null, result);
+        return callback(null, resStdout.length == 0 ? null : resStdout);
```

Comment:
I tried to put here:

    return callback(resStderr.length == 0 ? null : resStderror, resStdout.length == 0 ? null : resStdout);

But null was never returned when everything went well, but "NaNNaN" instead. Any hints?